AnKi/Shaders/Include/ClusteredShadingTypes.h

```diff
@@ -145,15 +148,16 @@ ANKI_SHADER_STATIC_ASSERT(sizeof(FogDensityVolume) == _ANKI_SIZEOF_FogDensityVol
 struct GlobalIlluminationProbe
 {
 	Vec3 m_aabbMin;
-	Vec3 m_aabbMax;
+	F32 m_halfTexelSizeU; ///< (1.0 / textureSize(texArr[textureIndex]).x) / 2.0
 
+	Vec3 m_aabbMax;
 	U32 m_textureIndex; ///< Index to the array of volume textures.
-	F32 m_halfTexelSizeU; ///< (1.0 / textureSize(texArr[textureIndex]).x) / 2.0
 
 	/// Used to calculate a factor that is zero when fragPos is close to AABB bounds and 1.0 at fadeDistance and less.
 	ANKI_RP F32 m_fadeDistance;
+	F32 m_padding0;
```

Comment:
Correct me if I'm wrong but that's not enough is it? The base alignment of the struct is rounded up to the alignment of Vec4. But the size of the struct in C++ is not multiple of sizeof(Vec4). I think it should be

```
Vec3 m_padding0;
ANKI_RP F32 m_fadeDistance;
```